apps/meteor/server/models/raw/FreeSwitchEvent.ts

```diff
@@ -0,0 +1,45 @@
+import type { IFreeSwitchEvent, RocketChatRecordDeleted } from '@rocket.chat/core-typings';
+import type { IFreeSwitchEventModel, InsertionModel } from '@rocket.chat/model-typings';
+import type { IndexDescription, Collection, Db, FindOptions, FindCursor, WithoutId } from 'mongodb';
+
+import { BaseRaw } from './BaseRaw';
+
+export class FreeSwitchEventRaw extends BaseRaw<IFreeSwitchEvent> implements IFreeSwitchEventModel {
+	constructor(db: Db, trash?: Collection<RocketChatRecordDeleted<IFreeSwitchEvent>>) {
+		super(db, 'freeswitch_events', trash);
+	}
+
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{ key: { channelUniqueId: 1, sequence: 1 }, unique: false },
```

Comment:
based on the `registerEvent` method, it looks like the idea is that these fields are unique.
```suggestion
			{ key: { channelUniqueId: 1, sequence: 1 }, unique: true },
```

Reply:
Since events from FS are a bit weird, think the `unique` false are there because some events may have same channel + seq and be different 🙈

Reply:
But let's try, worst case a try catch may be enough.

Reply:
but the code below is _ignoring_ events with the same `channelUniqueId` and `sequence`: 
https://github.com/RocketChat/Rocket.Chat/blob/0d685216e30d93fe10a77f7b7b0b5e5b183f8ea8/apps/meteor/server/models/raw/FreeSwitchEvent.ts#L23-L25

so that is basically an "unique index logic", but it doesn't make sense taking that responsibility if the database can take of that.. also moving that responsibility to mongo is safer.